pytest_splunk_addon/splunk.py

```diff
@@ -1014,3 +1031,6 @@ def pytest_unconfigure(config):
             os.remove(PYTEST_XDIST_TESTRUNUID + "_events")
         if os.path.exists(PYTEST_XDIST_TESTRUNUID + "_events.lock"):
             os.remove(PYTEST_XDIST_TESTRUNUID + "_events.lock")
+        if os.path.exists(PYTEST_XDIST_TESTRUNUID + "_ingestion_error"):
+            sleep(20)
```

Comment:
Why are we waiting 20s before deleting the file? any idea?

Reply:
Okay, but either of the hooks pytest_unconfigure or session_finish would run only after the test run has  finished, so Ideally it should work without sleep IMO.
 

> the file was removed, before they could check it
The file would only be removed after the test execution would be finished right?

Reply:
Let me double check then. Yes, I'd like to remove this file after test are finished.